frankfurt/nucleus-dev/web-deploy.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     app: nucleus-dev
     type: web
-  name: nucleus-dev
+  name: nucleus-dev-web
```

Comment:
@duallain any idea if this will cause issues? Wasn't sure about this one, but it's how basket is named.

Reply:
as long as service/deployment match, it shouldn't be an issue.

Reply:
here's the service: https://github.com/mozmeao/nucleus-config/blob/master/frankfurt/nucleus-dev/svc.yaml#L14-L16
the selector would have to change to match `nucleus-dev-web`

Reply:
Would it? The `app:` labels match in both cases. But the `app:` in the service selector needs to match the `name:` from the deployment?

Reply:
ahh, you're right pmac.  Name/selector being disjoint is perfectly fine, just wasn't always the convention before.

Reply:
Yeah. Basket uses `app: basket-dev` in the selector in the service so I think we're all good. Thanks y'all!